com.darhon.syncbackup.yml

```
@@ -0,0 +1,55 @@
+app-id: com.darhon.syncbackup
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: syncbackup.sh
+finish-args:
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --filesystem=host
+modules:
+  - name: rsync
+    buildsystem: autotools
+    config-opts:
+      - --disable-zstd
+      - --disable-xxhash
+      - --disable-md2man
+    sources:
+      - type: git
+        url: https://github.com/WayneD/rsync.git
+        tag: v3.2.3
+        commit: e94bad1c156fc3910f24e2b3b71a81b0b0bdeb70
+    cleanup:
+      - /share/man
+
+  - name: ksshaskpass
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://github.com/KDE/ksshaskpass.git
+        tag: v5.23.4
+        commit: 17771f575050f4b21cc04522df0fc2d7f7013f44
+    cleanup:
+      - /share/man
+
+  - name: syncbackup
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      post-install:
```

```suggestion
    post-install:
```

(this is the indentation problem mentionned below